packages/fluent/scss/colorgradient/_layout.scss

```diff
@@ -0,0 +1,184 @@
+        }
+
+        &.k-slider-horizontal {
+            width: var( --kendo-colorgradient-slider-horizontal-size, #{$kendo-colorgradient-slider-horizontal-size} );
+            height: var( --kendo-colorgradient-slider-track-size, #{$kendo-colorgradient-slider-track-size} );
+            flex: 0 0 var( --kendo-colorgradient-slider-track-size, #{$kendo-colorgradient-slider-track-size} );
+        }
+        &.k-slider-horizontal .k-slider-track {
+            height: var( --kendo-colorgradient-slider-track-size, #{$kendo-colorgradient-slider-track-size} );
+            border-radius: var( --kendo-colorgradient-slider-border-radius, #{$kendo-colorgradient-slider-border-radius} );
+            margin-top: calc( ( var( --kendo-colorgradient-slider-track-size, #{$kendo-colorgradient-slider-track-size} ) / 2 ) * -1 );
+        }
+
+        &.k-alpha-slider .k-slider-track::before {
+            content: "";
+            width: 100%;
+            height: 100%;
+            display: block;
+            border-radius: 2px;
+            position: relative;
+            z-index: -1;
+        }
+
+        .k-slider-track {
+            border-width:  var( --kendo-colorgradient-slider-border-width, #{$kendo-colorgradient-slider-border-width} );
+            border-style: solid;
+        }
+
+        .k-slider-wrap,
+        .k-slider-track-wrap {
+            z-index: 1;
+        }
+
+        .k-slider-selection {
+            display: none;
+        }
+
+    }
+
+    // Inputs
+    .k-colorgradient-inputs {
+        gap: var( --kendo-colorgradient-input-spacing, #{$kendo-colorgradient-input-spacing} );
+
+        .k-colorgradient-input-label {
+            text-transform: uppercase;
+        }
+
+        > .k-vstack {
+            gap: var( --kendo-colorgradient-input-label-spacing, #{$kendo-colorgradient-input-label-spacing} );
+        }
+
+        .k-numerictextbox {
+            width: var( --kendo-colorgradient-input-width, #{$kendo-colorgradient-input-width} );
+        }
+    }
+
+    // Contrast
+    .k-colorgradient-color-contrast {
+
+        > div {
+            display: flex;
+            flex-flow: row nowrap;
+            gap: var( --kendo-colorgradient-contrast-spacing, #{$kendo-colorgradient-contrast-spacing} );
+        }
+
+        .k-contrast-ratio {
+            margin-bottom: var( --kendo-colorgradient-contrast-spacing, #{$kendo-colorgradient-contrast-spacing} );
+        }
+
+        .k-contrast-ratio-text {
+            font-weight: var( --kendo-colorgradient-contrast-ratio-font-weight, #{$kendo-colorgradient-contrast-ratio-font-weight} );
+        }
+
+        .k-contrast-validation {
+            display: inline-flex;
+            align-items: center;
+            gap: calc( var( --kendo-colorgradient-contrast-spacing,  #{$kendo-colorgradient-contrast-spacing} ) / 2 );
+
+            .k-icon {
+                vertical-align: middle;
```

Comment:
do we need the vertical-align: middle of the icon here since the parent has align-items: center?

Reply:
needed for the stacking arrows